src/main/java/com/blamejared/crafttweaker/impl/commands/crafttweaker/MiscCommands.java

```diff
@@ -81,6 +81,16 @@ public static void registerMiscCommands(final TriConsumer<LiteralArgumentBuilder
             return 0;
         }));
         
+        CTCommands.registerCommand(CTCommands.playerCommand("wiki", "Opens a link to the wiki", (player, stack) -> {
+            PacketHandler.CHANNEL.send(PacketDistributor.PLAYER.with(() -> (ServerPlayerEntity) player), new MessageOpen("https://docs.blamejared.com/1.16/en/index"));
```

Comment:
We are totally going to forget to update this with MC versions, so best to just send them to the base page right?

```suggestion
            PacketHandler.CHANNEL.send(PacketDistributor.PLAYER.with(() -> (ServerPlayerEntity) player), new MessageOpen("https://docs.blamejared.com"));
```